GitCommands/AsyncLoader.cs

```diff
@@ -79,8 +79,7 @@ public Task Load(Action loadContent, Action onLoaded)
         public Task Load(Action<CancellationToken> loadContent, Action onLoaded)
         {
             Cancel();
-            if (_cancelledTokenSource != null)
-                _cancelledTokenSource.Dispose();
+            _cancelledTokenSource?.Dispose();
```

Comment:
:memo: Conversions of null check to `?.` should be sent as a single pull request with no other changes